app/assets/javascripts/custom.js

```diff
@@ -104,27 +121,54 @@ $(document).ready(function(){
     });
   })
 
+  $(".add_to_sale").click(function(){
+     var food_id = $(this).data("message");
+     var sale_price = $(".sale_price_"+ food_id).html();
```

Comment:
Identifier 'sale_price' is not in camel case.<br>Identifier 'food_id' is not in camel case.